### ISSUE TITLE
Rhythm doctor

### ISSUE THREAD

**Issue by meowmeowmoew3** [Wamy-Dev/ReziWebsite#21]
The game isn't available on the search engine and i don't wanna dl from igg

**Comment by Wamy-Dev** [comment]
This is likely because this is a new indie game and isn't as well known, and hasn't been cracked yet. Maybe try cs.rin.ru instead. Haven't checked there myself (most downloads are blocked on cs.rin.ru anyways thanks to google drive).

If igggames are the only ones who "have" the game and gog doesnt even have it, then the game hasn't been "pirated" and therefor can't download without guaranteeing a virus on your system.

In short, its not available anywhere, so don't download it.

**Comment by meowmeowmoew3** [comment]
I downloaded it from from cs.Rin.ru and it works so i think it has been cracked? I don't know

**Comment by Wamy-Dev** [comment]
> I downloaded it from from cs.Rin.ru and it works so i think it has been cracked? I don't know

I guess cracked is the wrong word. If it's available on cs.rin.ru then the only reason its not on any of the sites I scrape is cause it isn't as popular. I have tried scraping cs.rin.ru but it requires an account to actually view and download forum pages, and I didn't want rezi to have downloads like that.
